inc/core/theme-block-patterns.php

```diff
@@ -124,19 +124,33 @@ function bcgov_blocks_theme_register_block_patterns() {
 		$block_patterns[] = 'cleanbc/cleanbc-home-cover-banner';
 		$block_patterns[] = 'cleanbc/cleanbc-info-cover-banner';
 		$block_patterns[] = 'cleanbc/cleanbc-landing-cover-banner';
+		$block_patterns[] = 'cleanbc/cleanbc-level-3-dark-banner';
```

Comment:
these are a lot of patterns, with not so clear indication semantic wise what they are for., observation.. I know sometimes this is difficult, but just keep this in mind, when trying to fix a pattern a month from now.

Reply:
Patterns have a preview generated in the admin along with their title. This aids with identification and modification.